packages/core/test/Form.test.jsx

```diff
@@ -3349,19 +3349,19 @@ describe('Form omitExtraData and liveOmit', () => {
       omitExtraData,
     });
 
-    sandbox.stub(comp.ref.current, 'getUsedFormData').returns({
+    sandbox.stub(comp.ref.current, 'omitExtraData').returns({
       foo: '',
     });
 
     fireEvent.change(node.querySelector('[type=text]'), {
       target: { value: 'new' },
     });
 
-    sinon.assert.notCalled(comp.ref.current.getUsedFormData);
+    sinon.assert.notCalled(comp.ref.current.omitExtraData);
   });
 
-  describe('getUsedFormData', () => {
-    it('should call getUsedFormData when the omitExtraData prop is true', () => {
+  describe('omitExtraData on submit', () => {
+    it('should call omitExtraData when the omitExtraData prop is true', () => {
```

Comment:
It may be useful to keep testing `getUsedFormData()` independently of `omitExtraData()`... @nickgros your thoughts?

Reply:
Looks like this is just verifying that it's called when the prop is set. Method is unit tested below